lib/surface_site_web/live/components_basics.ex

```diff
@@ -9,14 +9,13 @@ defmodule SurfaceSiteWeb.ComponentsBasics do
   def render(assigns) do
     ~H"""
     <div style="position: relative;">
-      <MobileSidebar/>
-      <div class="sidebar-bg"/>
+      <MobileSidebar />
+      <div class="sidebar-bg" />
       <div class="container is-fullhd">
         <section class="main-content columns">
           <Sidebar />
           <div class="container column" style="background-color: #fff;">
             <div class="section" style="padding-bottom: 0px;">
-
               <nav class="breadcrumb" aria-label="breadcrumbs">
```

Comment:
The formatter takes away this newline because it has a rule that no extra newlines be added between opening/closing tags and their children.

E.g.

```html
<div>

  <span>Hello</span>

</div>
```

becomes

```html
<div>
  <span>Hello</span>
</div>
```

I'm curious how people feel about this.

Reply:
I feel totally fine with that :)